prow/plugins/golint/golint.go

```diff
@@ -233,11 +234,21 @@ func handle(minimumConfidence float64, ghc githubClient, gc *git.Client, log *lo
 	var comments []github.DraftReviewComment
 	for f, ls := range nps {
 		for l, p := range ls {
+			var suggestion = suggestion.SuggestCodeChange(p)
 			var body string
 			if p.Link == "" {
-				body = fmt.Sprintf("Golint %s: %s. %s", p.Category, p.Text, commentTag)
+				if suggestion != "" {
+					body = fmt.Sprintf("%s\nGolint %s: %s. %s", suggestion, p.Category, p.Text, commentTag)
+				} else {
+					body = fmt.Sprintf("Golint %s: %s. %s", p.Category, p.Text, commentTag)
+				}
 			} else {
-				body = fmt.Sprintf("Golint %s: %s. [More info](%s). %s", p.Category, p.Text, p.Link, commentTag)
+				if suggestion != "" {
+					body = fmt.Sprintf("%s\nGolint %s: %s. [More info](%s). %s", suggestion, p.Category,
+						p.Text, p.Link, commentTag)
+				} else {
+					body = fmt.Sprintf("Golint %s: %s. [More info](%s). %s", p.Category, p.Text, p.Link, commentTag)
+				}
 			}
```

Comment:
The branches of the `if p.Link == "" {` conditional are almost identical. They can be combined if you first determine if you handle the link separately and just use an empty value if the link is not included. Similarly, if `SuggestCodeChange` ensures that the output has a `\n` suffix if there is a suggestion then we can further simplify to get:
```go
var link string
if p.Link != "" {
  link = fmt.Sprintf("[More info](%s). ", p.Link)
}
...
body = fmt.Sprintf("%sGolint %s: %s. %s%s", suggestion, p.Category, p.Text, link, commentTag)
```